datatypes-jws/src/commonMain/kotlin/at/asitplus/crypto/datatypes/jws/JsonWebKey.kt

```diff
@@ -105,36 +112,79 @@ data class JsonWebKey(
         keyId ?: "urn:ietf:params:oauth:jwk-thumbprint:sha256:${jwkThumbprint}"
     }
 
-    override fun toString(): String {
-        return "JsonWebKey(type=$type, curve=$curve, keyId=$keyId," +
-                " x=${x?.encodeToString(Base64Strict)}," +
-                " y=${y?.encodeToString(Base64Strict)})"
-    }
+    override fun toString() =
+        when (type) {
+            JwkType.EC -> "JsonWebKey(" +
```

Comment:
I would print all members in `toString()` methods, regardless of `type`

Reply:
Changed